packages/zoe/test/unitTests/contracts/test-priceAggregator.js

```diff
@@ -1020,6 +1020,40 @@ test('storage keys', async t => {
 
   t.is(
     await subscriberSubkey(E(publicFacet).getSubscriber()),
-    'paTest:mockChainStorageRoot.priceAggregator.ATOM_USD_price_feed',
+    'fake:mockChainStorageRoot.priceAggregator.ATOM_USD_price_feed',
+  );
+});
+
+test('storage', async t => {
+  const { zoe, makeFakePriceOracle, makeMedianAggregator } = t.context;
+  const aggregator = await makeMedianAggregator(1n);
+  const { timer: oracleTimer } = await E(zoe).getTerms(aggregator.instance);
+
+  const price1000 = await makeFakePriceOracle(1000n);
+  await E(aggregator.creatorFacet).initOracle(price1000.instance, {
+    increment: 10n,
+  });
+  await oracleTimer.tick();
+  t.deepEqual(
+    t.context.mockStorageRoot.getBody(
+      'mockChainStorageRoot.priceAggregator.ATOM_USD_price_feed',
+    ),
+    {
+      quoteAmount: {
+        brand: { iface: 'Alleged: quote brand' },
+        value: [
+          {
+            amountIn: { brand: { iface: 'Alleged: $ATOM brand' }, value: 1n },
```

Comment:
`1n` doesn't seem right; shouldn't that be `1_000_000n`?

Reply:
In dev chains, yes. In this test the brand has no decimal places:
https://github.com/Agoric/agoric-sdk/blob/c81ad73e616c998239610e62751b2ac036b14d26/packages/zoe/test/unitTests/contracts/test-priceAggregator.js#L107

I mentioned this in test plan,
>  I tried setting the decimalPlaces in the $ATOM issuer (as it must be on chain) but that broke many other test expressions.

What coverage are you looking for?

Reply:
Took some refactoring 092d8f4209292df0ac02cdda17b18f40d0fdc55c